src/components/Auth/Login/Login.js

```diff
@@ -45,12 +45,14 @@ const styles = {
   inputStyle: {
     height: '35px',
     marginBottom: '10px',
+    webkitTextFillColor: 'unset',
```

Comment:
I believe this is only for chrome and safari? Please add the relevant css for other browsers.